app/src/main/java/com/kickstarter/viewmodels/HorizontalRewardViewHolderViewModel.kt

```diff
@@ -256,6 +254,10 @@ interface HorizontalRewardViewHolderViewModel {
 
         }
 
+        private fun pledgeCopyIsVisible(project: Project, reward: Reward): Boolean {
```

Comment:
I think this function would be more descriptive if called something like `rewardIsAvailable(project: reward:)`, because it gets easier to understand what's going on here:
```
this.minimumAmount = this.projectAndReward
                 .filter { rewardIsAvailable(it.first, it.second) }
                 .map { this.ksCurrency.format(it.second.minimum(), it.first) }
```